packages/styles/stories/Components/avatar/avatar.stories.js

```diff
@@ -54,7 +54,7 @@ const localStyles = `
 <style>
 
     .fd-avatar {
-        margin: 0 0.5rem;
+        margin: 0.5rem 0.5rem;
```

Comment:
When the value is the same in horizontal and vertical direction you can just use:
```
margin: 0.5rem;
```